hal/targets/cmsis/TARGET_Freescale/TARGET_MCU_K64F/TOOLCHAIN_GCC_ARM/MK64FN1M0xxx12.ld

```diff
@@ -52,9 +52,11 @@ ENTRY(Reset_Handler)
 
 __ram_vector_table__ = 1;
 
-/* Heap 1/4 of ram and stack 1/8 */
-__stack_size__ = 0x8000;
-__heap_size__ = 0x10000;
+/* Heap 1/4 of ram and stack 1/8. 0x1000 is reduced from both heap
```

Comment:
The comment `` Heap 1/4 of ram and stack 1/8`` is out of date with this change. Also, the heap size was decreased by 0x7000 bytes, not 0x1000. Finally, these values don't actually set the heap or stack size, it just reserves space for it. The interrupt stack is the last 1K, and the heap is the rest.